core/chain_makers.go

```diff
@@ -577,7 +577,6 @@ func GenerateVerkleChain(config *params.ChainConfig, parent *types.Block, engine
 
 func GenerateVerkleChainWithGenesis(genesis *Genesis, engine consensus.Engine, n int, gen func(int, *BlockGen)) (common.Hash, ethdb.Database, []*types.Block, []types.Receipts, []*verkle.VerkleProof, []verkle.StateDiff) {
 	db := rawdb.NewMemoryDatabase()
-	saveVerkleTransitionStatusAtVerlkeGenesis(db)
```

Comment:
Duplicated with the one in `genesis.Commit(db, triedb)`. It's also not correct to persist a transition-state with zero root.